web/frontend/package.json

```diff
@@ -40,6 +57,8 @@
     "@viamrobotics/rpc": "0.1.37",
     "@viamrobotics/sdk": "0.5.0-rc.0",
     "@viamrobotics/three": "^0.0.2",
+    "@viamrobotics/eslint-config": "^0.2.5",
+    "@viamrobotics/prettier-config": "^0.3.3",
     "@viamrobotics/typescript-config": "^0.0.4",
     "cypress": "12.17.3",
     "eslint": "8.46.0",
```

Comment:
- Missing explicit dependencies:
  - `eslint-config-prettier`
  - `eslint-plugin-sonarjs`
- No longer needed dependencies:
  - `eslint-import-resolver-custom-alias`
  - `eslint-plugin-import`
  - `eslint-plugin-promise`

Reply:
Thanks for the catch! Fixed